server/events.go

```diff
@@ -2679,6 +2691,42 @@ func (s *Server) nsubsRequest(sub *subscription, c *client, _ *Account, subject,
 	s.sendInternalMsgLocked(reply, _EMPTY_, nil, nsubs)
 }
 
+type kickClientReq struct {
+	Name string `json:"name"`
+	Id   uint64 `json:"id"`
```

Comment:
```suggestion
	ID   uint64 `json:"id"`
```

Reply:
Changed it to ID for both kick and ldm requests types